src/test/java/nl/tudelft/contextproject/audio/AudioManagerTest.java

```diff
@@ -39,19 +39,12 @@ public void tearDown() {
 	 */
 	@Test
 	public void testRegisterVolume() {
-		//Register the volume
 		AudioManager.getInstance().registerVolume(an, SoundType.AMBIENT);
-		
-		//Registering should set the volume to the correct type.
 		verify(an, times(1)).setVolume(SoundType.AMBIENT.getGain());
-		
-		//Reset the mock
+
 		reset(an);
-		
-		//Change the gain
+
 		SoundType.AMBIENT.setGain(2.0f);
-		
-		//The volume should have been updated
 		verify(an, times(1)).setVolume(2.0f);
```

Comment:
While obvious, I think the comments here do give a step by step overview of what is expected to happen after something is done.

I feel that (at least some) comment should be readded, but I'm curious how other team members think about this.